app/App.tsx

```diff
@@ -46,12 +48,22 @@ class App extends React.PureComponent<Props, State> {
     super(props);
     this.state = {
       announcements: true,
+      addthisLoaded: false,
     };
   }
 
+  handleScriptLoad() {
+    this.setState({ addthisLoaded: true });
```

Comment:
Doesn't matter really here, but try to use the setState variant that accepts a function. It's a touch safer so it's just a good habit to get into.

So in this case, it would be this:

```ts
this.setState(state => ({ ...state, addthisLoaded: true }));
```